### PR TITLE
Temporary swallow SocketException in DthTest

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

**Comment by Sridhar-MS** [comment]
you should print this https://msdn.microsoft.com/en-us/library/system.net.sockets.socketexception.errorcode(v=vs.110).aspx.

File: test/dotnet-projectmodel-server.Tests/DthTestClient.cs [hunk 1]
```diff
@@ -225,7 +225,17 @@ public void Dispose()
...
+            try
+            {
+                _socket.Shutdown(SocketShutdown.Both);
+            }
+            catch (SocketException)
+            {
+                // Swallow this error for now.
+                // This is a temporary fix for a random failure on CI. The issue happens on Windowx x86
+                // only.
+            }
```
Comment (Sridhar-MS) [inline test/dotnet-projectmodel-server.Tests/DthTestClient.cs]: Should you call `_socket.Close()` after shutting down?

**Comment by troydai** [comment]
@dotnet-bot test this please

**Comment by troydai** [comment]
@Sridhar-MS Windows x86 passed. Looking good?

**Comment by Sridhar-MS** [comment]
:shipit:

**Comment by troydai** [comment]
Somehow one of the build is rerunning. I'll wait till it finishes,

**Pull request merged**